api/src/main/java/org/openmrs/validator/UserValidator.java

```diff
@@ -193,4 +196,17 @@ public boolean isUserNameAsEmailValid(String username) {
 		Matcher matcher = EMAIL_PATTERN.matcher(username);
 		return matcher.matches();
 	}
+	
+	/**
+	 * @return true if email is valid or null
+	 * @param email
+	 */
+	public boolean isEmailValid(String email) {
+		if (StringUtils.isBlank(email)) {
+			return true;
```

Comment:
If it's blank then why do you return true?

Reply:
Null is accepted so preventing the validation from failing i let it know it should allow null but if something is supplied then validate with email pattern.

Reply:
This is a utility method an I guess which should return null if email is blank, the code that calls this should first check if not blank before calling it